src/main/java/com/minecolonies/coremod/proxy/ClientProxy.java

```diff
@@ -252,6 +254,10 @@ public static void registerModels(@NotNull final ModelRegistryEvent event)
             ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(frame), 0,
                         new ModelResourceLocation(frame.getRegistryName(), INVENTORY));
         }
+
+        //Additionally we register an exclusion handler here;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)